### ISSUE TITLE
A JavaScript error occurred in the main process for Len 4.0.0

### ISSUE THREAD

**Issue by wangdaqian08** [lensapp/lens#1687]
**Describe the bug**
A JavaScript error occurred in the main process

Uncaught Exception:
TypeError: Cannot read property 'initMainWindow' of undefined
    at App.<anonymous> (/Applications/Lens.app/Contents/Resources/app.asar/static/build/main.js:1:153533)
    at App.emit (events.js:223:5)

**To Reproduce**
1. Download Lens 4.0.0
2. click Lens from Application

**Expected behavior**
Launch Lens 

**Screenshots**
<img width="414" alt="Screen Shot 2020-12-08 at 17 50 06" src="https://user-images.githubusercontent.com/10279340/101449988-df996780-397d-11eb-982a-ab88b76ca5f8.png">

**Environment (please complete the following information):**
- Lens Version:4.0.0
- OS: 10.15.7
- Installation method (download from git release ):

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
Your logs go here...
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
your kubeconfig here
```

**Additional context**

**Comment by Mollet-Tristan** [comment]
Hi, I have the same issue on MacOs Big Sur

<img width="299" alt="error_lens" src="https://user-images.githubusercontent.com/34402702/101467530-b8717480-3942-11eb-8b44-564acfabdb33.png">

**Comment by stephenpope** [comment]
Same error here too (MacOS Big Sur)

**Edit**...

Fixed by this in a related issue: https://github.com/lensapp/lens/issues/1567#issuecomment-739219895

**Comment by Mollet-Tristan** [comment]
Thanks work for me too

**Comment by wangdaqian08** [comment]
> Same error here too (MacOS Big Sur)
> 
> **Edit**...
> 
> Fixed by this in a related issue: [#1567 (comment)](https://github.com/lensapp/lens/issues/1567#issuecomment-739219895)

Thanks, @stephenpope  I have tried these steps,but not working for me. Because I can't find the folder, I searched for the location, but it doesn't exist.

I uninstalled and reinstalled a few times for the 4.0.0 and 3.6.x versions.
4.0.0 shows the error like the screenshot in this thread.
3.6.x doesn't show error at all, but no user interface either, only see Lens in the dock.
